apis/discovery/v1alpha1/foreigncluster_types.go

```diff
@@ -198,6 +200,7 @@ type TenantNamespaceType struct {
 // +kubebuilder:printcolumn:name="Outgoing peering",type=string,JSONPath=`.status.peeringConditions[?(@.type == 'OutgoingPeering')].status`
 // +kubebuilder:printcolumn:name="Incoming peering",type=string,JSONPath=`.status.peeringConditions[?(@.type == 'IncomingPeering')].status`
 // +kubebuilder:printcolumn:name="Networking",type=string,JSONPath=`.status.peeringConditions[?(@.type == 'NetworkStatus')].status`
+// +kubebuilder:printcolumn:name="API Server",type=string,JSONPath=`.status.peeringConditions[?(@.type == 'ForeignAPIServerStatus')].status`
```

Comment:
I would add a "priority=1" annotation (as for the cluster id and cluster name) to avoid too wide outputs